tests/test_a11y.py

```diff
@@ -282,17 +282,6 @@ def test_notebook_ipywidget_output_tab_stop(page: Page, url_base: str) -> None:
     assert ipywidget.evaluate("el => el.tabIndex") == 0
 
 
-def test_breadcrumb_expansion(page: Page, url_base: str) -> None:
```

Comment:
Removed this as we seem to have complete test in `test_playwright`

Reply:
Yep that appears to be cruft